ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -285,21 +249,37 @@ def jedi2fv3inc(self: Analysis) -> None:
         # Reference the python script which does the actual work
         incpy = os.path.join(self.task_config.HOMEgfs, 'ush/jediinc2fv3.py')
 
+        # create template dictionaries
+        template_inc = self.task_config.COM_ATMOS_ANALYSIS_TMPL
+        tmpl_inc_dict = {
+            'ROTDIR': self.task_config.ROTDIR,
+            'RUN': self.runtime_config.RUN,
```

Comment:
`task_config` should have this attribute.
```suggestion
            'RUN': self.task_config.RUN,
```

Reply:
`self.runtime_config` replaced with `self.task_config`.   Done at [`4402e25`](https://github.com/RussTreadon-NOAA/global-workflow/commit/4402e255ee53b5059c5cfdad8364a80101fd6b08).   Thank you @aerorahul .  I'm still learning what all these different `self.*` mean and when/where we should one or the other.